src/main/app/initialize.test.js

```diff
@@ -202,11 +202,13 @@ describe('main/app/initialize', () => {
     });
 
     describe('initializeAfterAppReady', () => {
-        it('should set spell checker URL if applicable', async () => {
-            Config.spellCheckerURL = 'http://server-1.com';
-            await initialize();
-            expect(session.defaultSession.setSpellCheckerDictionaryDownloadURL).toHaveBeenCalledWith('http://server-1.com/');
-        });
+        if (process.platform !== 'darwin') {
+            it('should set spell checker URL if applicable', async () => {
+                Config.spellCheckerURL = 'http://server-1.com';
+                await initialize();
+                expect(session.defaultSession.setSpellCheckerDictionaryDownloadURL).toHaveBeenCalledWith('http://server-1.com/');
+            });
+        }
```

Comment:
is this change expected? seems to test a different thing. Looks ok, but not sure if you wanted to add that change as well.